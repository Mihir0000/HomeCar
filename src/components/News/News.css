.news_section {
    width: 100%;
    float: left;
    padding-bottom: 90px;
}

.news_section_2 {
    width: 100%;
    float: left;
}

.layout_padding {
    padding-top: 90px;
    padding-bottom: 0px;
}
.health_taital {
    width: 100%;
    float: left;
    font-size: 40px;
    color: #252525;
    font-weight: bold;
}

.health_text {
    width: 100%;
    float: left;
    font-size: 16px;
    color: #252525;
    margin: 0px;
}

.box_main {
    width: 100%;
    float: left;
    background-color: #252525;
    height: auto;
    border-radius: 15px;
    padding: 40px;
}

.box_main:hover {
    background-color: #1becde;
}

.box_main.active {
    background-color: #1becde;
}

.icon_1 {
    width: 100%;
    float: left;
    text-align: center;
    padding-bottom: 20px;
    line-height: 95px;
}

.daily_text {
    width: 40%;
    margin: 0 auto;
    text-align: center;
    font-weight: bold;
    font-size: 24px;
    display: block;
    color: #fff;
    line-height: 30px;
}

.daily_text_1 {
    width: 40%;
    margin: 0 auto;
    text-align: center;
    font-weight: bold;
    font-size: 24px;
    display: block;
    color: #fff;
    line-height: 30px;
    padding-bottom: 40px;
}


.getquote_bt {
    width: 180px;
    margin: 0 auto;
    text-align: center;
}

.getquote_bt a {
    width: 100%;
    float: left;
    font-size: 18px;
    color: #ffffff;
    background-color: #1becde;
    padding: 7px 0px;
    text-align: center;
    margin-top: 50px;
}

.getquote_bt a:hover {
    background-color: #252525;
}