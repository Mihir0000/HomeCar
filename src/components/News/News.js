import React from 'react';
import { Card, Col, Container, Image, Row } from 'react-bootstrap';
import './News.css';

const News = ({ time }) => {
    return (
        <Container fluid className="news_section layout_padding">
            <Card className="container bg-transparent border-0">
                <Card.Title className="health_taital fs-1 pb-2">
                    Why choose {time} home care
                </Card.Title>
                <Card.Text className="health_text pb-3">
                    labore et dolore magna aliqua. Ut enim ad minim veniam
                </Card.Text>
                <Card.Body className="news_section_2 layout_padding">
                    <Row>
                        <Col lg={4} sm={6}>
                            <Card.Subtitle className="box_main">
                                <Card.Subtitle className="icon_1">
                                    <Image
                                        style={{ width: '50px' }}
                                        src="static/img/icon-2.png"
                                        alt="icon"
                                    />
                                </Card.Subtitle>
                                <Card.Text className="daily_text fs-4">
                                    Daily care experts
                                </Card.Text>
                            </Card.Subtitle>
                        </Col>
                        <Col lg={4} sm={6}>
                            <Card.Subtitle className="box_main active">
                                <Card.Subtitle className="icon_1">
                                    <Image
                                        style={{ width: '55px' }}
                                        src="static/img/icon-3.png"
                                        alt="icon"
                                    />
                                </Card.Subtitle>
                                <Card.Text className="daily_text_1 fs-4">
                                    Available 24/7
                                </Card.Text>
                            </Card.Subtitle>
                        </Col>
                        <Col lg={4} sm={6}>
                            <Card.Subtitle className="box_main">
                                <Card.Subtitle className="icon_1">
                                    <Image
                                        style={{ width: '40px' }}
                                        src="static/img/icon-4.png"
                                        alt="icon"
                                    />
                                </Card.Subtitle>
                                <Card.Text className="daily_text_1 fs-4">
                                    Balanced care
                                </Card.Text>
                            </Card.Subtitle>
                        </Col>
                    </Row>
                </Card.Body>
                <Card.Footer className="getquote_bt bg-transparent border-0">
                    <Card.Link
                        href="/#"
                        className="d-flex justify-content-between"
                    >
                        Get A Quote
                        <Card.Subtitle className="d-flex align-self-center m-2">
                            <Image
                                src="static/img/right-arrow.png"
                                alt="right-arrow"
                            />
                        </Card.Subtitle>
                    </Card.Link>
                </Card.Footer>
            </Card>
        </Container>
    );
};

export default News;
