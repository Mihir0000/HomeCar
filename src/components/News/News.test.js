import React from 'react';
import { render } from '@testing-library/react';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer';
import News from './News';

test('snapshot testing --news', () => {
    const component = renderer.create(<News />);
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
});

// crash test --news
it('render without crashing --news', () => {
    const div = document.createElement('div');
    render(<News />, div);
    ReactDOM.unmountComponentAtNode(div);
});
