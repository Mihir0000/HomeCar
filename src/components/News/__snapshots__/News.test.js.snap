// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot testing --news 1`] = `
<div
  className="news_section layout_padding container-fluid"
>
  <div
    className="container bg-transparent border-0 card"
  >
    <div
      className="health_taital fs-1 pb-2 card-title h5"
    >
      Why choose 
       home care
    </div>
    <p
      className="health_text pb-3 card-text"
    >
      labore et dolore magna aliqua. Ut enim ad minim veniam
    </p>
    <div
      className="news_section_2 layout_padding card-body"
    >
      <div
        className="row"
      >
        <div
          className="col-lg-4 col-sm-6"
        >
          <div
            className="box_main card-subtitle h6"
          >
            <div
              className="icon_1 card-subtitle h6"
            >
              <img
                alt="icon"
                className=""
                src="static/img/icon-2.png"
                style={
                  Object {
                    "width": "40px",
                  }
                }
              />
            </div>
            <p
              className="daily_text_1 fs-5 card-text"
            >
              Daily care experts
            </p>
          </div>
        </div>
        <div
          className="col-lg-4 col-sm-6"
        >
          <div
            className="box_main active card-subtitle h6"
          >
            <div
              className="icon_1 card-subtitle h6"
            >
              <img
                alt="icon"
                className=""
                src="static/img/icon-3.png"
                style={
                  Object {
                    "width": "55px",
                  }
                }
              />
            </div>
            <p
              className="daily_text_1 fs-5 card-text"
            >
              Available 24/7
            </p>
          </div>
        </div>
        <div
          className="col-lg-4 col-sm-6"
        >
          <div
            className="box_main card-subtitle h6"
          >
            <div
              className="icon_1 card-subtitle h6"
            >
              <img
                alt="icon"
                className=""
                src="static/img/icon-4.png"
                style={
                  Object {
                    "width": "38px",
                  }
                }
              />
            </div>
            <p
              className="daily_text_1 fs-5 card-text"
            >
              Balanced care
            </p>
          </div>
        </div>
      </div>
    </div>
    <div
      className="getquote_bt bg-transparent border-0 card-footer"
    >
      <a
        className="d-flex justify-content-between card-link"
        href="/#"
      >
        Get A Quote
        <div
          className="d-flex align-self-center m-2 card-subtitle h6"
        >
          <img
            alt="right-arrow"
            className=""
            src="static/img/right-arrow.png"
          />
        </div>
      </a>
    </div>
  </div>
</div>
`;
